tests/trustyai/utils.py

```diff
@@ -0,0 +1,84 @@
+from typing import List
+
+from ocp_resources.maria_db import MariaDB
+from ocp_resources.mariadb_operator import MariadbOperator
+from ocp_resources.pod import Pod
+from ocp_utilities.operators import TIMEOUT_5MIN
+from timeout_sampler import TimeoutSampler, TimeoutExpiredError
+from simple_logger.logger import get_logger
+
+
+LOGGER = get_logger(name=__name__)
+
+
+def wait_for_mariadb_operator_pods(mariadb_operator: MariadbOperator, timeout: int = 300) -> None:
+    def _check_if_mariadb_operator_pods_ready() -> bool:
+        expected_pods: List[str] = [
+            "mariadb-operator",
+            "mariadb-operator-cert-controller",
+            "mariadb-operator-helm-controller-manager",
+            "mariadb-operator-webhook",
+        ]
+
+        pods = Pod.get(namespace=mariadb_operator.namespace)
+
+        for pod_prefix in expected_pods:
+            matching_pods = [pod for pod in pods if pod.name.startswith(pod_prefix)]
+
+            if matching_pods:
```

Comment:
```suggestion
            if matching_pods := [pod for pod in pods if pod.name.startswith(pod_prefix)]:

```

Reply:
Changed this function to wait for deployment replicas